### PR TITLE
use name of env created to get config where engine will be installed

### PR DESCRIPTION
**Pull request by ca7alindev**

### CODE REVIEW

**Comment by evghen1** [comment]
@ca7alindev Solution not working, return error:
```
$ go run cmd/kndp/main.go env create local
INFO Creating environment with Kubernetes engine 'kind'
FATA context "kind-local" does not exist
exit status 1
```

**Review by evghen1 — CHANGES_REQUESTED** [review]
Please test solution.

File: internal/environment/kind.go [hunk 1]
```diff
@@ -56,8 +56,8 @@ func KindEnvironment(ctx context.Context, context string, logger *log.Logger, na
 	stderrScanner := bufio.NewScanner(stderr)
 	for stderrScanner.Scan() {
 		line := stderrScanner.Text()
-		if !strings.Contains(line, " • ") {
-			logger.Print(line)
+		if strings.Contains(line, "ERROR") {
+			logger.Fatalf("%s: host port %d already in use", line, hostPort)
```
Comment (evghen1) [inline internal/environment/kind.go:60]: @ca7alindev Ahahaha... Do you think just single type of error will happens?

**Review by evghen1 — CHANGES_REQUESTED** [review]

File: internal/environment/kind.go (hunk 1 above)
Comment (ca7alindev) [reply internal/environment/kind.go:60]: @evghen1 , no but at least we need to print errors if exists

Comment (evghen1) [reply internal/environment/kind.go:60]: @ca7alindev You print  all errors as already in use port, not? Exists other errors.

Comment (ca7alindev) [reply internal/environment/kind.go:60]: @evghen1 then i will let just the error printed without other custom description?

**Review by evghen1 — APPROVED** [review]

**Pull request merged**